lstchain/scripts/onsite/onsite_create_calibration_file.py

```diff
@@ -162,7 +156,7 @@ def main():
             os.system(cmd)
 
             # plot and save some results
-            plot_file=f"{output_dir}/log/calibration.Run{run}.{sub_run}.pedestal.Run{ped_run}.0000.pdf"
+            plot_file=f"{output_dir}/log/calibration.Run{run:05d}.{sub_run:04d}.pedestal.Run{ped_run:04d}.0000.pdf"
```

Comment:
Why is run sometimes formatted with 5 digits and sometimes with only 4?

Reply:
this is probably a typo and should:
`Run{ped_run:04d}.0000.pdf` -> `Run{ped_run:05d}.0000.pdf`

after this I think we should be good to go with this pR as well